tests/openvpn/test_backend.py

```diff
@@ -882,3 +882,31 @@ def test_tls_auth_key_present(self):
         )
         client = OpenVpn(client_config)
         self.assertEqual(client.render(), self._openvpn_client_tls_auth_render)
+
+    def test_ca_same_file_path_for_same_device(self):
+        conf = {
+            "cert": "cert.pem",
+            "dev": "tap0",
+            "dev_type": "tap",
+            "dh": "dh.pem",
+            "key": "key.pem",
+            "mode": "server",
+            "name": "test-1",
+            "proto": "udp",
+            "tls_server": True,
+        }
+        template1 = {
+            "openvpn": [{"ca": "TEST", **conf}],
```

Comment:
For realisticness, make "ca" point to `/etc/x509/ca.pem`.

**The configuration of the two VPN templates must be different between the 2 templates otherwise the validation error is righlty raised.** 

There's no point in trying to apply 2 identical templates!